Library/Homebrew/formula_versions.rb

```diff
@@ -32,11 +33,12 @@ def file_contents_at_revision(rev)
   end
 
   def formula_at_revision(rev)
+    yield @formula_at_revision[rev] if @formula_at_revision[rev]
     contents = file_contents_at_revision(rev)
```

Comment:
It's early so I'm not sure I quite understand how/why an extra `yield` is in here and it's not passing the contents. Can you elaborate?